pkg/kubelet/stats/cri_stats_provider_windows.go

```diff
@@ -53,17 +54,19 @@ func (s networkStats) GetHNSEndpointStats(endpointName string) (*hnslib.HNSEndpo
 func (p *criStatsProvider) listContainerNetworkStats() (map[string]*statsapi.NetworkStats, error) {
 	networkStatsProvider := newNetworkStatsProvider(p)
 
+	ctx := context.TODO()
```

Comment:
It would be nice to have an explanation comment for every usage of `context.TODO()` explaining that this is a temporary solution and when it must be changed to a permanent one.

Reply:
Thanks, updated.

Reply:
The same here - please pass context from the upper level, thanks!